include/groonga/util.h

```diff
@@ -214,6 +214,80 @@ grn_inspect_name(grn_ctx *ctx, grn_obj *buffer, grn_obj *obj);
  */
 GRN_API grn_obj *
 grn_inspect_encoding(grn_ctx *ctx, grn_obj *buffer, grn_encoding encoding);
+/**
+ * \brief Inspect the given type and produce its name.
+ *
+ * \since 4.0.8
+ *
+ * For example usage:
+ * ```c
+ * grn_obj type_name;
+ * GRN_TEXT_INIT(&type_name, 0);
+ * grn_inspect_type(ctx, &type_name, obj->header.type);
+ * printf("#=> %.*s\n", (int)GRN_TEXT_LEN(&type_name),
+ * GRN_TEXT_VALUE(&type_name)); GRN_OBJ_FIN(ctx, &type_name);
```

Comment:
```suggestion
 * printf("%.*s\n", (int)GRN_TEXT_LEN(&type_name), GRN_TEXT_VALUE(&type_name));
 * GRN_OBJ_FIN(ctx, &type_name);
```

I think it would be better without `#=>` to match the output example.

Reply:
fix: [ea80160](https://github.com/groonga/groonga/pull/2366/commits/ea80160252f13a72a63b10d7156fc06ca1483808) Thanks!